parser/src/module.rs

```diff
@@ -266,11 +266,28 @@ fn parse_module_name(input: &str) -> IResult<&str, &str> {
 ///
 /// returns: impl Parser<&str, ModuleArguments, Error<&str>>+Sized
 ///
-fn get_module_args_parser<'a>(
-    inline: bool,
-) -> impl Parser<&'a str, ModuleArguments, Error<&'a str>> {
+fn get_module_args_parser<'a>(inline: bool) -> impl Parser<&'a str, MaybeArgs, Error<&'a str>> {
     map(
         opt(alt((
+            map(
+                tuple((
+                    separated_list0(
+                        get_arg_separator_parser(inline),
+                        get_unnamed_arg_parser(inline),
+                    ),
+                    many0(get_arg_separator_parser(inline)),
```

Comment:
I don't think this `many0` is needed since `get_arg_separator_parser` is `space1`/`multispace1` and thus catches as many spaces as possible (we can't reapply it when it already caught all spaces)